.appveyor.yml

```diff
@@ -53,9 +53,9 @@ build_script:
 
 test_script:
   - if "%APPVEYOR_SCHEDULED_BUILD%"=="True" (
-      gradlew --stacktrace -Pkotlin.parallel.tasks.in.project=false dokkaJar check
+      gradlew --stacktrace dokkaJar check
```

Comment:
Commit message: Could you please add a link to https://youtrack.jetbrains.com/issue/KT-29525?

Reply:
Done.